ui/pages/confirmations/components/confirm/splash/smart-account-update/smart-account-update.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { useState } from 'react';
+import { NameType } from '@metamask/name-controller';
+import { TransactionMeta } from '@metamask/transaction-controller';
+
+import { SMART_ACCOUNT_INFO_LINK } from '../../../../../../../shared/lib/ui-utils';
+import {
+  Box,
+  Button,
+  ButtonSize,
+  ButtonVariant,
+  Text,
+} from '../../../../../../components/component-library';
+import {
+  AlignItems,
+  BackgroundColor,
+  BlockSize,
+  BorderRadius,
+  Display,
+  FlexDirection,
+  FontWeight,
+  JustifyContent,
+  TextColor,
+  TextVariant,
+} from '../../../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../../../hooks/useI18nContext';
+import Name from '../../../../../../components/app/name';
+import { useConfirmContext } from '../../../../context/confirm';
+import { useSmartAccountActions } from '../../../../hooks/useSmartAccountActions';
+
+export function SmartAccountUpdate() {
+  const [acknowledged, setAcknowledged] = useState(false);
+
+  const t = useI18nContext();
+  const { currentConfirmation } = useConfirmContext<TransactionMeta>();
+  const { chainId, txParams } = currentConfirmation ?? {};
+  const { from } = txParams;
+
+  const { handleRejectUpgrade } = useSmartAccountActions();
+
+  if (!currentConfirmation || acknowledged) {
+    return null;
+  }
+
+  return (
+    <Box
+      display={Display.Flex}
+      backgroundColor={BackgroundColor.overlayDefault}
+      color={TextColor.primaryDefault}
+      className="smart-account-update__wrapper"
+    >
+      <Box
+        backgroundColor={BackgroundColor.backgroundDefault}
+        borderRadius={BorderRadius.MD}
+        display={Display.Flex}
+        flexDirection={FlexDirection.Column}
+        alignItems={AlignItems.center}
+        justifyContent={JustifyContent.spaceBetween}
+        margin={4}
+        padding={4}
+        className="smart-account-update__inner"
+      >
+        <img
+          width="100%"
+          src="./images/smart-transactions/smart-account-update.svg"
+        />
+        <Text fontWeight={FontWeight.Medium} variant={TextVariant.headingLg}>
+          {t('smartAccountSplashTitle')}
+        </Text>
+        <Box display={Display.Flex} alignItems={AlignItems.center}>
+          <Text
+            color={TextColor.textAlternative}
+            variant={TextVariant.bodyMd}
+            marginInlineEnd={2}
+          >
+            {t('smartAccountRequestFor')}
+          </Text>
+          <Name
+            value={from}
+            type={NameType.ETHEREUM_ADDRESS}
+            preferContractSymbol
+            variation={chainId}
+          />
+        </Box>
+        <Box display={Display.Flex} alignItems={AlignItems.flexStart}>
```

Comment:
Minor, could we make a local component for this like `ListItem` to avoid the duplication?

Reply:
Addressed in [PR](https://github.com/MetaMask/metamask-extension/pull/31774)